pyfr/writers/vtk.py

```diff
@@ -57,6 +57,10 @@ def __init__(self, args):
                         (f'grad {var} {f}', nf(f)) for f in fields
                     )
 
+    @staticmethod
+    def _vtkfile_version():
```

Comment:
No need for a method here.  Can just assign an attribute in the class definition.